How to/Library Bounds to Viewer Bounds/PdfViewerSample/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,83 @@
+﻿@using Syncfusion.EJ2
+@{
+    ViewBag.Title = "Home Page";
+}
+
+<div>
+    <div style="height:500px;width:1350px;">
+        <br /><br />
+        @Html.EJS().PdfViewer("pdfviewer").DocumentPath("https://cdn.syncfusion.com/content/pdf/pdf-succinctly.pdf").ServiceUrl("https://services.syncfusion.com/js/production/api/pdfviewer").Render()
```

Comment:
@SF4524LogeshKumar Why we have used production service URL in the MVC sample. Please check and resolve it.